server/models/db/worker.js

```diff
@@ -11,6 +11,21 @@ async function getWorker(email, password) {
     return userModel.getUser(email, password, Tables.worker, Worker);
 }
 
+// This method inserts a worker account given specific information.
+async function insertWorkerAccount(firstName, lastName, type, workerID, email, password, phone, specialization) {
```

Comment:
Hi Nancy,
Minor change but I would also recommend changing `workerID` to `workerId` to increase variable naming consistency.

Reply:
Hi Amy,
Thank you for suggesting that!
My action: I've changed the name from workerID to workerId.
You can now see these changes the following locations:

line 15 of server/models/db>worker.js
line 18 of server/models/db>worker.js
line 19 of server/models/db>worker.js

Please confirm if this takes care of your suggestion!

Reply:
Hi Amy,
I have also made similar changes to studentId. 
My action: I've changed the name from studentID to studentId.
You can now see these changes the following locations:

line 14 of server/models/db>student.js
line 17 of server/models/db>student.js
line 18 of server/models/db>student.js

Reply:
Hi Nancy,
Perfect, highly appreciate it!